src/FondOfSpryker/Zed/ThirtyFiveUpApi/Business/Model/ThirtyFiveUpApiInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace FondOfSpryker\Zed\ThirtyFiveUpApi\Business\Model;
+
+use Generated\Shared\Transfer\ApiCollectionTransfer;
+use Generated\Shared\Transfer\ApiDataTransfer;
+use Generated\Shared\Transfer\ApiItemTransfer;
+use Generated\Shared\Transfer\ApiRequestTransfer;
+
+interface ThirtyFiveUpApiInterface
+{
+    /**
+     * @param int $idThirtyFiveUpOrder
+     * @param \Generated\Shared\Transfer\ApiDataTransfer $apiDataTransfer
+     *
+     * @return \Generated\Shared\Transfer\ApiItemTransfer
+     */
+    public function update(int $idThirtyFiveUpOrder, ApiDataTransfer $apiDataTransfer): ApiItemTransfer;
+
+    /**
+     * @param \Generated\Shared\Transfer\ApiRequestTransfer $apiRequestTransfer
+     *
+     * @throws \Spryker\Zed\Api\Business\Exception\ApiDispatchingException
```

Comment:
phpcs?!

Reply:
vendor/bin/phpcbf --standard=vendor/spryker/code-sniffer/Spryker/ruleset.xml vendor/fond-of-spryker/thirty-five-up-api

No fixable errors were found

Time: 542ms; Memory: 14MB

Reply:
Cool